CFC_WebApp/clients/choice/choice.py

```diff
@@ -43,13 +44,28 @@ def getResult(user_uuid):
   from dao.user import User
   from dao.client import Client
 
+  user_uuid = "6433c8cf-c4c5-3741-9144-5905379ece6e"
   user = User.fromUUID(user_uuid)
-  renderedTemplate = template("clients/choice/result_template.html",
-                          variables = json.dumps({'curr_view': getCurrView(user_uuid),
-                                       'uuid': str(user_uuid),
-                                       'client_key': Client("choice").getClientKey()}),
-                          gameResult = base64.b64encode(gamified.getResult(user_uuid)),
-                          dataResult = base64.b64encode(default.getResult(user_uuid)))
+
+  # renderedTemplate = template("clients/choice/result_template.html",
+  #                         variables = json.dumps({'curr_view': getCurrView(user_uuid),
+  #                                      'uuid': str(user_uuid),
+  #                                      'client_key': Client("choice").getClientKey()}),
+  #                         gameResult = base64.b64encode(gamified.getResult(user_uuid)),
+  #                         dataResult = base64.b64encode(default.getResult(user_uuid)),
+  #                         recommendationResult = base64.b64encode(recommendation.getResult(user_uuid)))
+  # return renderedTemplate
+
+  # DEMO
+  originalTrip = get_trip_db().find_one({'user_id': UUID(user.uuid), 'recommended_alternative': {'$exists': True} })
```

Comment:
@jimmydief, I can't merge this and deploy to the real server - showing a hardcoded trip from a particular user is not going to work. Also, other teams have their own screens. You need to fix this the right way before midnight tomorrow. Reverting now...